Update documentation (#63)
yiisoft/yii-queue: docs/guide/worker.md

Configuration
================

> In order to use implemented worker, you should resolve it's dependencies (e.g. through DI container) 

Reviewer: ```suggestion
In order to use implemented worker, you should resolve its dependencies (e.g. through DI container) 
```

and define handlers for each message which will be consumed by this worker;

> Handlers are callables indexed by payload names. When a message will be consumed from the queue, a callable associated with it's payload name will be called.

Reviewer: ```suggestion
Handlers are callables indexed by payload names. When a message is consumed from the queue, a callable associated with its payload name is called.
```

For more info about Supervisor's configuration and usage see its [documentation](http://supervisord.org).

Note that worker daemons started with `queue/listen` are only supported by the 

> [AMQP Interop](https://github.com/yiisoft/yii-queue-amqp) driver. For additional options see driver guide.

Reviewer 2: I don't think we should say smthng like that. All the drivers must implement `DriverInterface` which has both `run` and `listen` methods.
